### PR TITLE
DOC-1102: Add information about the `system` database

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-1102

### CODE REVIEW

**Review by stbof — APPROVED** [review]
Would be nice to reference a supported database in the pre-loaded databases paragraph.

File: _includes/v21.1/sql/preloaded-databases.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+New clusters and existing clusters [upgraded](upgrade-cockroach-version.html) to v2.1 or later will include auto-generated databases, with the following purposes:
+
+- The empty `defaultdb` database is used if a client does not specify a database in the [connection parameters](connection-parameters.html).
+- The `movr` database contains data about users, vehicles, and rides for the vehicle-sharing app [MovR](movr.html).
+- An empty database called `postgres` is provided for compatibility with PostgreSQL client applications that require it.
```
Comment (stbof) [inline _includes/v21.1/sql/preloaded-databases.md:5]:
```suggestion
- The empty `postgres`database is provided for compatibility with PostgreSQL client applications that require it.
```

File: _includes/v21.2/sql/preloaded-databases.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+New clusters and existing clusters [upgraded](upgrade-cockroach-version.html) to v2.1 or later will include auto-generated databases, with the following purposes:
+
+- The empty `defaultdb` database is used if a client does not specify a database in the [connection parameters](connection-parameters.html).
+- The `movr` database contains data about users, vehicles, and rides for the vehicle-sharing app [MovR](movr.html).
+- An empty database called `postgres` is provided for compatibility with PostgreSQL client applications that require it.
```
Comment (stbof) [inline _includes/v21.2/sql/preloaded-databases.md:5]:
```suggestion
- The empty `postgres` database is provided for compatibility with PostgreSQL client applications that require it.
```

File: _includes/v21.1/sql/preloaded-databases.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+New clusters and existing clusters [upgraded](upgrade-cockroach-version.html) to v2.1 or later will include auto-generated databases, with the following purposes:
```
[the remark this answers is not in the document]
Comment (nickvigilante) [reply _includes/v21.1/sql/preloaded-databases.md:1]: I think if it's true to say CRDB v2.1, then it's also true to say whatever the page version it is on, so I'll use `{{ page.version.version }}` here instead.

**Review by stbof — APPROVED** [review]

**Pull request merged**